### PR TITLE
Savas   lesson13

### PR DESCRIPTION
**Pull request by savaspamuk**

### CODE REVIEW

File: responsive-grid-layout/savas/global.css
Comment (annasvst) [inline responsive-grid-layout/savas/global.css:1]: Hi Savaş, 3 files from another lesson (responsive-grid-layout) are added to this PR. What I think has happened is you created a branch for this PR while still being on the branch you worked before, not on the "main". This is an error that happens very often so the most important thing when creating a new branch is to check that you are on the "main" branch or that you're creating a new branch from "main" branch.

Now, these 3 files need to be removed from this PR. To do that in the current situation, there are a few ways but the simplest would be to create a new branch from "main" locally, and then manually copy the content of the file here to a correct file: https://github.com/frontend-from-0/240123-exercises/pull/98/commits/cd26c69a1d62cb4cb1630135241c03a81afe8518

Comment (savaspamuk) [reply responsive-grid-layout/savas/global.css:1]: Hi Anna, I created a new branch and a pull request. I hope it is ok now :)

Comment (annasvst) [reply responsive-grid-layout/savas/global.css:1]: The new PR looks good, I'll close this one :)

**Comment by annasvst** [comment]
Closing this PR in favor of https://github.com/frontend-from-0/240123-exercises/pull/99/files

**Pull request closed without merging**